esp/src/eclwatch/WsWorkunits.js

```diff
@@ -32,9 +32,14 @@ define([
             service: "WsWorkunits",
             action: "WUShowScheduled",
             responseQualifier: "WUShowScheduledResponse.Workunits.ScheduledWU",
-            idProperty: "Wuid"
-    });
+            idProperty: "calculatedID",
 
+        preProcessRow: function (row) {
+            lang.mixin(row, {
+                calculatedID: row.Wuid + row.EventText
+            });
```

Comment:
Minor Comment:  ```row.calculatedID = row.Wuid + row.EventText;``` would have been equally good (but you may have needed to check if "row" existed).